### ISSUE TITLE
QAQC being coded incorrectly in pull from WQReview compared to database

### ISSUE THREAD

**Issue by lrking0530** [USGS-R/WQ-Review#186]
Data was pulled and a QAQC sample was coded as a blank in the file from WQReview but was coded as a replicate in the database. It was the same record number, date, time, etc. After about 4 tries WQReview pulled it correctly.
[Replicate.xlsx](https://github.com/USGS-R/WQ-Review/files/11140658/Replicate.xlsx)
[Blank.xlsx](https://github.com/USGS-R/WQ-Review/files/11140659/Blank.xlsx)
![QWdata_1](https://user-images.githubusercontent.com/129780944/229577232-9175a7ba-f270-4717-8f52-d59fc8cd9a6a.JPG)

**Comment by cpenn-usgs** [comment]
Hi @lrking0530 , WQReview is not doing any sample medium coding itself, it's pulling the medium code directly from what is in the NWIS DB at the time of access. Is it possible the sample was being worked on, reloaded, or edited in the database while you were retrieving it and there was a lag?

**Comment by lrking0530** [comment]
No, it was an older sample and everything in the record was approved. No one else did anything with the sample record.

Lindsey King
Hydrologist
United States Geological Survey
Wyoming-Montana Water Science Center
3162 Bozeman Ave
Helena, MT 59601
***@***.***
Cell phone: (406) 465-7021
________________________________
From: Colin Penn ***@***.***>
Sent: Wednesday, April 12, 2023 12:39:18 PM
To: USGS-R/WQ-Review ***@***.***>
Cc: King, Lindsey R ***@***.***>; Mention ***@***.***>
Subject: [EXTERNAL] Re: [USGS-R/WQ-Review] QAQC being coded incorrectly in pull from WQReview compared to database (Issue #186)




 This email has been received from outside of DOI - Use caution before clicking on links, opening attachments, or responding.



Hi @lrking0530<https://github.com/lrking0530> , WQReview is not doing any sample medium coding itself, it's pulling the medium code directly from what is in the NWIS DB at the time of access. Is it possible the sample was being worked on, reloaded, or edited in the database while you were retrieving it and there was a lag?
